.gitignore

```diff
@@ -7,3 +7,4 @@
 config.json
 /app/scripts/config.js
 .DS_Store
+/app/fonts/fa-*
```

Comment:
_This one weird trick will make 2000 lines of code disappear from your git repository!_

Reply:
I think the removal of 2000 lines of code is not related to this change, it's related to running the `npm install` again with `node v14` and committing that updated lock file.

The excluded fonts are the new `fontawesone` files which were never added before.

Reply:
Ah okay! It was just a joke though, about those ads about "this one weird trick will ..." 😉

Reply:
Ok, that is "new" for me!